.gitignore

```diff
@@ -1,6 +1,7 @@
-bin/
+/bin/
```

Comment:
Shouldn't this be bin/

Reply:
No this will cause all bin folders to be excluded even from vendor/**